index.html

```diff
@@ -4,6 +4,17 @@
     <meta charset="utf-8">
     <meta name="description" content="CSS Editor">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
```

Comment:
```suggestion
    <style> This is my first hackathon and mt first-time code reviewing.  First of all congratulations on making it this far!! Like you, I also used plain JS, HTML, and my only main suggestion is to separate the styling separately in a CSS.style instead of inside the index.html  
```

Reply:
I agree.
Separating your styles would do a world of wonder!
It also makes code reviews easier.